### DISCUSSION TITLE
Implementing Node based UI

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#1634] [Showcase]
Now you can make a node-based UI in tkinter using `TkNodeSystem!`

# [TkNodeSystem](https://github.com/Akascape/TkNodeSystem)

![Example App](https://github.com/Akascape/TkNodeSystem/assets/89206401/ea818333-c979-4402-bc7c-8850930dc087)
![node_menu](https://github.com/Akascape/TkNodeSystem/assets/89206401/0ba5ba42-4787-4b94-8b40-682084df2e80)

**Comment by hawkhero2** [comment]
you really like this project :). Looks neat
